src/app/games/lasca/lasca.component.ts

```diff
@@ -62,25 +64,30 @@ export class LascaComponent extends ParallelogramGameComponent<LascaRules,
     private currentMoveClicks: Coord[] = [];
     private capturedCoords: Coord[] = []; // Only the coords capture by active player during this turn
     private legalMoves: LascaMove[] = [];
+    private moveGenerator: LascaMoveGenerator = new LascaMoveGenerator();
 
     public constructor(messageDisplayer: MessageDisplayer) {
         super(messageDisplayer);
         this.hasAsymmetricBoard = true;
         this.rules = LascaRules.get();
         this.node = this.rules.getInitialNode();
-        this.availableMinimaxes = [
-            new LascaControlMinimax('Lasca Control Minimax'),
-            new LascaControlAndDominationMinimax(),
+        this.availableAIs = [
+            new Minimax('Control', this.rules, new LascaControlHeuristic(), new LascaMoveGenerator()),
```

Comment:
this.moveGenerator ? (x3)

Reply:
abawé, je savais pas qu'il existait lui !